Bugzilla/Config/Auth.pm

```diff
@@ -107,18 +107,41 @@ sub get_param_list {
         },
 
         {
-            name    => 'password_complexity',
```

Comment:
`password_complexity` is still used by `account/email/confirm-new.html.tmpl` and `global/password-features.html.tmpl`.